pieria/patterns/sidebar.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Title: sidebar
+ * Slug: pieria/sidebar
+ * Categories: hidden
+ * Inserter: no
+ */
+?>
+<!-- wp:group {"style":{"spacing":{"blockGap":"3rem","padding":{"top":"0px","right":"0px","bottom":"0px","left":"0px"},"margin":{"top":"0px","bottom":"0px"}}},"layout":{"type":"flex","orientation":"vertical"}} -->
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0px;padding-right:0px;padding-bottom:0px;padding-left:0px"><!-- wp:group {"style":{"spacing":{"blockGap":"0.5rem","padding":{"top":"0rem","right":"0rem","bottom":"0rem","left":"0rem"},"margin":{"top":"0px","bottom":"0px"}}},"layout":{"type":"default"}} -->
+<div class="wp-block-group" style="margin-top:0px;margin-bottom:0px;padding-top:0rem;padding-right:0rem;padding-bottom:0rem;padding-left:0rem"><!-- wp:paragraph -->
+<p>Sappho, spelled (in the dialect spoken by the poet) Psappho, (born c. 610, Lesbos, Greece — died c. 570 BCE). A lyric poet greatly admired in all ages for the beauty of her writing style.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph -->
+<p>Her language contains elements from Aeolic vernacular and poetic tradition, with traces of epic vocabulary familiar to readers of Homer. She has the ability to judge critically her own ecstasies and grief, and her emotions lose nothing of their force by being recollected in tranquillity.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:group -->
+
+<!-- wp:image {"id":145,"sizeSlug":"full","linkDestination":"none","style":{"color":{"duotone":["#111111","#ffbbbb"]}}} -->
+<figure class="wp-block-image size-full"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/sappho-statue_bw.png" alt="" class="wp-image-145"/></figure>
+<!-- /wp:image -->
+
+<!-- wp:paragraph {"align":"center"} -->
+<p class="has-text-align-center">Designed with <a rel="nofollow" href="https://wordpress.org">WordPress</a></p>
```

Comment:
This file contains some strings that should be translatable.

Reply:
Added  translation for "Designed with WordPress". I'm _guessing_ that the rest of the content doesn't need to be translatable, since it's a placeholder.